python/src/scipp/plot/plot_2d.py

```diff
@@ -91,95 +89,76 @@ def __init__(self,
                          aspect=aspect,
                          button_options=['X', 'Y'])
 
-        self.members.update({"images": {}, "colorbars": {}})
+        self.members["images"] = {}
         self.extent = {"x": [1, 2], "y": [1, 2]}
         self.logx = logx
         self.logy = logy
         self.vminmax = {"vmin": vmin, "vmax": vmax}
         self.global_vmin = np.Inf
         self.global_vmax = np.NINF
+        if resolution is not None:
+            if isinstance(resolution, int):
+                self.image_resolution = {"x": resolution, "y": resolution}
+            else:
+                self.image_resolution = resolution
+        else:
+            self.image_resolution = 0.64 * config.plot.dpi / 96.0
```

Comment:
What are these magic numbers?

Reply:
:+1: 
this is a remnant from experimentation. i will change back to using the config width and height.